tests/test_clerk_provider_unit.py

```
@@ -327,6 +345,26 @@ def test_wrap_app_allows_clerk_frontend_version_overrides():
+    app = rx.App()
+    wrap_app(
+        app,
+        publishable_key="pk_test",
+        localization={
+            "unstable__errors": {
+                "too_many_requests": "Too many sign-in attempts. Try again shortly.",
+            }
+        },
+    )
+
+    component = app.app_wraps[(1, "ClerkProvider")](False)
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)